app/proguard-rules.pro

```diff
@@ -17,56 +17,32 @@
 # keep setters in Views so that animations can still work.
 # see http://proguard.sourceforge.net/manual/examples.html#beans
 -keepclassmembers public class * extends android.view.View {
-   void set*(***);
-   *** get*();
+    void set*(***);
+    *** get*();
 }
 # We want to keep methods in Activity that could be used in the XML attribute onClick
 -keepclassmembers class * extends android.app.Activity {
-   public void *(android.view.View);
+    public void *(android.view.View);
 }
 # For enumeration classes, see http://proguard.sourceforge.net/manual/examples.html#enumerations
 -keepclassmembers enum * {
     public static **[] values();
     public static ** valueOf(java.lang.String);
 }
 -keepclassmembers class * implements android.os.Parcelable {
-  public static final android.os.Parcelable$Creator CREATOR;
-}
--keepclassmembers class **.R$* {
```

Comment:
This one is the only rule that was risky to remove, but I tested it and it seems save. Gets ~100kb off the apk size.

Reply:
I'm not sure for which fields it is

Reply:
Its for the fields of the generated R file. Seems like some tools look up fields via reflection so it was in the default proguard config